src/DynamoCoreWpf/ViewModels/Search/SearchViewModel.cs

```diff
@@ -880,9 +880,15 @@ private void UpdateTopResult(SearchMemberGroup memberGroup)
             }
 
             var topMemberGroup = new SearchMemberGroup(memberGroup.FullyQualifiedName);
-            topMemberGroup.AddMember(memberGroup.Members.First());
 
+            // Clone top node.
+            var topNode = new NodeSearchElementViewModel(memberGroup.Members.FirstOrDefault());
```

Comment:
As you pointed out, when user moves the selection to `ByCoordinates` it also highlights the *Top Result* right? That sounds *okay* to me. Do we intentionally clone an item just to get rid of the highlight? That sounds *overkill* to me, the existing behavior sounds just *good enough* for me, cloning does not buy us much at the cost of complicating the code, do we really want to do this? @Racel @lillismith, what do you guys think of the double-highlighting effect?